### DISCUSSION TITLE
Unable to GitHub Education.

### DISCUSSION THREAD

**Discussion by asika-2005** [community/community#46954] [GitHub Education]
# Issue
I try to register GitHub Education Program.
But it fails because of following errors.
```ruby:qitta.rb
Your application cannot be reviewed until you fix the following:
You are significantly more likely to be verified if you have completed your <a href="https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile">GitHub user profile</a> with your full name and a short bio.
The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.
The image you submitted does not appear to match the proof type you selected. If your original document is not in English, please take a photo of the original next to an English translation.
```
# What I want to do
I'd like to join GitHub Education.
# What I did
I sent some of document such as School ID and Portal web site.
In addition I asked to GitHub support but denied.

**Comment by SilasRodrigues19** [comment]
Hey @asika-2005 

I found these 2 discussions that may help you

#44112
#21118

At this link here you can find some more discussions that have been marked as answered related to GitHub Student

[GitHub Education is:answered](https://github.com/orgs/community/discussions?discussions_q=GitHub+Education+is%3Aanswered)

**Comment by samus-aran** [comment]
Hi @asika-2005 👋 

Thank you for supplying your rejection reasons! Let's work through them and provide more context.

> You are significantly more likely to be verified if you have completed your GitHub user profile with your full name and a short bio.

Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

> The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.

Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps.

> The image you submitted does not appear to match the proof type you selected. If your original document is not in English, please take a photo of the original next to an English translation.

Is the text in your evidence English? If not then it's recommended that you translate it into English and display that translation along side your evidence. 

Hope this all helps. Let us know how you get on! 👍
